### ISSUE TITLE
ON2

### ISSUE THREAD

**Issue by fragandi** [TeamBasedInquiryLearning/linear-algebra#394]
Compute the orthogonal projection of Euclidean vector onto a line.

**Comment by jbunn3** [comment]
For @fragandi  to review:

Consider the following linear subspace of R^3: W=span(one vector in R^3)

Task 1: Explain and demonstrate how to find the orthogonal projection \vec u_W of \vec u=... onto W. 

Task 2: Explain and demonstrate how to find the orthogonal projection of \vec u_{W^\perp} of \vec u onto W^\perp.

Task 3: Express the orthogonal decomposition of \vec u with respect to W.

**Comment by fragandi** [comment]
I think this structure makes sense, are we sure we do not want to do the projection onto a 2-dimensional subspace so that one can test using the projection in two different directions?
Also if the complement is a line, the situation is easier to picture and they could even decide to try to compute the projection on the complement directly (if they do not see the easy version to do task 2).
 
I worry that otherwise the problem can be too straightforward as for task 2 the answer would be just u - ans from task 1, and task 3 is just ans1 + ans2.

**Comment by jbunn3** [comment]
> I worry that otherwise the problem can be too straightforward as for task 2 the answer would be just u - ans from task 1, and task 3 is just ans1 + ans2.

I agree that tasks 2 and 3 are straightforward as your described, but that's basically what I had done in the past and was OK with. That main concern I had was whether we wanted to introduce the theorem that (col A)^\perp = nul(A^T), which is used in the general case.

> Also if the complement is a line, the situation is easier to picture and they could even decide to try to compute the projection on the complement directly (if they do not see the easy version to do task 2).

I had thought about a similar idea to this as a task, which is a good one, I think.
